src/main/resources/static/assets/js/sdh.js

```diff
@@ -73,8 +75,84 @@
 		    _isShowDetails(field) {
 		    	return field.key === 'show_details';
 		    },
+		    _isEdit(field) {
+		    	return field.key === 'edit';
+		    },
 		    _formatDate(dateString) {
 		    	return dateString.replace(/ \d\d:\d\d:\d\d\.\d/, '');
+		    },
+		    edit(item, index, target) {
+		    	this.modalEditRow.domain = this.domainOptions.filter(it => it.description === item.domain)[0].id;
+		    	if (item.id === "") {
+		    		this.modalEditRow.title = 'New Clinical Review';
+		    	} else {
+		    		this.modalEditRow.title = 'Edit Clinical Review';
+		    		this.modalEditRow.id = item.id;
+		    	      this.modalEditRow.prioritySelected = this.modalEditRow.priorityOptions.filter(it => it.text === item.clinician_priority)[0].value;
+			        this.modalEditRow.readinessSelected = this.modalEditRow.readinessOptions.filter(it => it.text === item.patient_readiness)[0].value;
+		    	}
+				this.$root.$emit('bv::show::modal', 'modalEditRow', target);
+		    },
+		    resetModal() {
+		        this.modalEditRow.title = '';
+		        this.modalEditRow.id = '';
+		        this.modalEditRow.domain = '';
+		        this.modalEditRow.prioritySelected = '';
+		        this.modalEditRow.readinessSelected = '';
+		    },
+		    handleOk(evt) {
+		        // Prevent modal from closing so we can validate
+		        evt.preventDefault();
+		        if (!this.modalEditRow.prioritySelected || !this.modalEditRow.readinessSelected) {
+		          alert('Please enter values');
+		        } else {
+		          this.handleSubmit();
+		        }
+		    },
+		    handleSubmit() {
+				const patient = this.$el.getAttribute('data-patient-id');
+				const contextPath = this.$el.getAttribute('data-context-path');
+				let clinicianReviewId = this.modalEditRow.id;
+				let {url, method, data} = '';
```

Comment:
This line initializes `url`, `method`, and `data` to `undefined` via destructuring. You could accomplish the same thing with just `let url, method, data;`. If you're trying to do multiple assignment, it's worth noting that ESLint recommends against it, because [multiple assignment can have surprising behavior](https://stackoverflow.com/questions/1758576/multiple-left-hand-assignment-with-javascript) due to scope.